lib/mission_control/jobs/console/helpers.rb

```diff
@@ -11,7 +11,7 @@ def jobs_help
     puts "You are currently connected to #{MissionControl::Jobs::Current.server}" if MissionControl::Jobs::Current.server
 
     puts "You can connect to a job server with"
-    puts "  connect_to <app_id>:<server_id>\n\n"
+    puts '  connect_to "<app_id>:<server_id>"\n\n'
```

Comment:
This change causes it to print \n\n instead of 2 new line characters. You need double quotes on the string for `\n` to mean new line.
<img width="398" alt="Screenshot 2024-01-31 at 9 44 16 AM" src="https://github.com/basecamp/mission_control-jobs/assets/5104186/73427815-4701-45c3-bce4-2c7e36915918">

```suggestion
    puts %(  connect_to "<app_id>:<server_id>"\n\n)
```

Reply:
Ah yep, thanks! I originally used `\"` but rubocop complained and wasn't paying attention when I swapped it around.

Reply:
I figured as much